<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chọn Khách Hàng</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>991</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Danh Sách Khách Hàng</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tìm Kiếm :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtSearch">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbTypeSearch">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Họ Tên</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Địa Chỉ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Số Điện Thoại</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mã KH</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Email</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tbDanhSachKhachHang">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>961</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubmit">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>500</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Xác Nhận</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>500</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Trở Về</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
